.changeset/large-tables-wonder.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/cli': patch
+---
+
+Add experimental support for optional `auth` app entry point.
```

Comment:
I guess same question here, it's an entrypoint that is supposed only have auth-related stuff, but its actual path is `/public`?